bin/rucio

```diff
@@ -1490,9 +1491,10 @@ def list_rules(args):
                           '%s[%d/%d/%d]' % (rule['state'], rule['locks_ok_cnt'], rule['locks_replicating_cnt'], rule['locks_stuck_cnt']),
                           rule['rse_expression'],
                           rule['copies'],
+                          rule['size'],
```

Comment:
I am concerned about the output of `list-rules` being already very long in terms of width. Also, understanding the magnitude from bytes takes effort. I think it will be preferable to replace this with `sizefmt(rule['size']) if rule['size'] is not None else 'N/A'` (and adapt the header accordingly).